### PR TITLE
Create launch file to automatically determine Arduino port and run se…

### LINKED ISSUE #259: Automatically identify Arduino port and create offboard comms launch file
## Background

Currently, to run the Arduino code, we must manually determine the port it is attached to (like /dev/ttyUSB0), and then run the script using rosrun with the port we had determined. This means we cannot automatically launch the robot using a single launch file, since it requires human input to determine this port. This is inefficient and causes delays when we forget to rerun this script.

## Task
The overall goal is to have a launch file that will automatically run the `serial_node.py` script with the port parameter determined. The determination of this port parameter will probably be in a separate script, with its output being piped into the launch file. This launch file for offboard_comms will then be included in the launch files in the execute package which will allow us to automatically start up all relevant nodes in the robot with a single launch file.

### PR DESCRIPTION
**Pull request by ericdjiang**
Closes #259

### CODE REVIEW

File: onboard/catkin_ws/src/offboard_comms/README.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ To actually get the code onto the Arduino, you need to install the newly generat
 You can then use the ROS message types in Arduino code.
 
 ## Testing offboard communication
-First set up the node on the computer that will talk to the Arduino. To do this, use `lsusb` and `sudo dmesg | grep tty` to determine the serial port on which the Arduino is connected. Then run (replace `/dev/ttyUSB0` with the serial port you find):
+First set up the node on the computer that will talk to the Arduino.
 ```
-rosrun rosserial_python serial_node.py /dev/ttyUSB0
+roslaunch serial.launch
```
Comment (muthuArivoli) [inline onboard/catkin_ws/src/offboard_comms/README.md:22]:
```suggestion
roslaunch offboard_comms serial.launch
```

File: onboard/catkin_ws/src/offboard_comms/launch/serial.launch [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<launch>
```
Comment (muthuArivoli) [inline onboard/catkin_ws/src/offboard_comms/launch/serial.launch:1]: Add the servo_wrapper.py to this launch file, might as well have it integrated with the rest of the system.

Also, add this launch file to one of the launch files in the execute package (maybe motion.launch?). Make sure to not run this when sim=True, however you wish to handle that.

**Review by muthuArivoli — CHANGES_REQUESTED** [review]
I fixed up the commands, they work when I test on my NUC. Three comments, two of them very trivial, the one critical one is adding this launch file to the motion.launch in the execute package, making sure that it is not run when we want simulation.

**Review by ShaanGondalia — APPROVED** [review]
LGTM

**Review by muthuArivoli — APPROVED** [review]
LGTM

**Pull request merged**